ballerina/Ballerina.toml

```diff
@@ -1,12 +1,12 @@
 [package]
 org = "ballerinax"
 name = "nats"
-version = "2.0.2"
+version = "2.1.0"
```

Comment:
This needs to be changed in gradle.properties

Reply:
Lets also run the build and update the toml files

Reply:
Done.

Reply:
Could you check if it is the correct toml file, that has been updated